live_address.gemspec

```
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = Dir["spec/**/*.rb"]
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'activesupport', '~> 4.1'
+  spec.add_dependency 'activesupport', '~> 5.2'
```

```suggestion
  spec.add_dependency 'activesupport', '>= 4.1'
```

There's nothing about this gem that requires Rails 5, so it would be better to just require the minimum version here.